velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2137,6 +2137,65 @@ TEST_F(DateTimeFunctionsTest, dateDiffTimestamp) {
           fromTimestampString("2018-02-28 10:00:00.500")));
 }
 
+TEST_F(DateTimeFunctionsTest, dateDiffTimestampWithTimezone) {
+  const auto dateDiff = [&](const std::string& unit,
+                            std::optional<int64_t> timestamp1,
+                            const std::optional<std::string>& timeZoneName1,
+                            std::optional<int64_t> timestamp2,
+                            const std::optional<std::string>& timeZoneName2) {
+    auto timestampWithTimezoneVector1 =
```

Comment:
nit: ts1

for brevity

Reply:
Thanks for the prompt review!! Do we have a guideline about the naming convention in Velox? I believe in Presto we always use full names.

Reply:
Yes, there is a general "do not abbreviate" rule, but we need to apply common sense as well. Here, the name makes it difficult to read and understand the code, hence, I suggested a rename.